calib:
  cam1:
    cloud:
      frame: cam_center
      frame_pub: world
      tx: -12.0
      ty: 30.0
      tz: -40.0
      rx: 3.0
      ry: -1.9
      rz: 0.0
  cam2:
    cloud:
      tx: 0.0
      ty: 0.0
      tz: -10.0
      rx: -0.3
      ry: -1.0
      rz: 0.0
  scene:
    cloud:
      tx: 0.0
      ty: 0.0
      tz: 0.0
      rx: 0.0
      ry: 0.0
      rz: 0.0